NEWS.md

```diff
@@ -1,6 +1,24 @@
 # HistogramZoo v1.4.0
 
-## Added
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+
+### Added
+- ModelFit class to allow pretty-printing of `fit_distributions` results
+
+### Changed
+
+### Fixed
+- Bugs associated with fitting truncated distributions
+
+## [1.4.0] - 2023-04-19
+
+### Added
```

Comment:
Can you put this all under a section `# HistogramZoo v1.4.2` ?

I think we can have one header for each minor version `vX.X` and increment patch version `vX.X.X` as we go.

Reply:
Done :)